apps/asap-server/test/controllers/events.test.ts

```diff
@@ -1,920 +1,643 @@
-      const emptyEvent = eventBase;
-      emptyEvent.flatData!.notes = null;
-      emptyEvent.flatData!.videoRecording = null;
-      emptyEvent.flatData!.presentation = null;
-      emptyEvent.flatData!.meetingMaterials = null;
-      emptyEvent.flatData!.notesPermanentlyUnavailable = null;
-      emptyEvent.flatData!.videoRecordingPermanentlyUnavailable = null;
-      emptyEvent.flatData!.presentationPermanentlyUnavailable = null;
-      emptyEvent.flatData!.meetingMaterialsPermanentlyUnavailable = null;
+      test('Should return permanentlyUnavailable details if enough time has passed and details are empty', async () => {
+        const eventsGraphqlResponse = getSquidexEventsGraphqlResponse();
+        eventsGraphqlResponse.queryEventsContentsWithTotal!.items![0]!.flatData.notes =
+          null;
+        eventsGraphqlResponse.queryEventsContentsWithTotal!.items![0]!.flatData.videoRecording =
+          null;
+        eventsGraphqlResponse.queryEventsContentsWithTotal!.items![0]!.flatData.presentation =
+          null;
+        eventsGraphqlResponse.queryEventsContentsWithTotal!.items![0]!.flatData.meetingMaterials =
+          null;
+
+        squidexGraphqlClientMock.request.mockResolvedValueOnce(
+          eventsGraphqlResponse,
+        );
 
-      nock(config.baseUrl)
-        .post(`/api/content/${config.appName}/graphql`, {
-          query: print(FETCH_EVENT),
-          variables: {
-            id: eventId,
-          },
-        })
-        .reply(200, { data: { findEventsContent: emptyEvent } });
-
-      const result = await events.fetchById(eventId);
-      expect(result).toEqual({
-        ...eventResponse,
-        notes: null,
-        videoRecording: null,
-        presentation: null,
-        meetingMaterials: null,
+        const result = await eventsController.fetch({ before: 'before' });
+
+        expect(result.items[0]?.notes).toBeNull();
+        expect(result.items[0]?.videoRecording).toBeNull();
+        expect(result.items[0]?.meetingMaterials).toBeNull();
+        expect(result.items[0]?.presentation).toBeNull();
       });
-    });
 
-    test('Should return empty (undefined) details if empty but the event is fresh', async () => {
-      const now = new Date().toISOString();
-      const emptyEvent = eventBase;
-      emptyEvent.flatData!.endDate = now;
-      emptyEvent.flatData!.notes = null;
-      emptyEvent.flatData!.videoRecording = null;
-      emptyEvent.flatData!.presentation = null;
-      emptyEvent.flatData!.meetingMaterials = null;
-      emptyEvent.flatData!.notesPermanentlyUnavailable = null;
-      emptyEvent.flatData!.videoRecordingPermanentlyUnavailable = null;
-      emptyEvent.flatData!.presentationPermanentlyUnavailable = null;
-      emptyEvent.flatData!.meetingMaterialsPermanentlyUnavailable = null;
+      test('Should return empty (undefined) details if empty but the event is fresh', async () => {
```

Comment:
again im not sure what 'fresh' is - maybe its a good time to make sure explain exactly what criteria needs to be met to return those empty details?